### ISSUE TITLE
high memory usage

### ISSUE THREAD

**Issue by disaster123** [EinEinfach/CaSSAndRA#90]
I've detected that cassandra needs a lot of memory - currently it uses 1.4GB of memory after some time on my linux docker-container. I remember that cassandra was using far less some time ago...

**Comment by EinEinfach** [comment]
RAM?

**Comment by disaster123** [comment]
Yes RAM. I don‘t think it’s related to docker. Anything I could check? May be purge config?

**Comment by EinEinfach** [comment]
What is your "max age for measured data" value? How is the size of measure folder?

**Comment by disaster123** [comment]
With an empty config it consumes 85MB in my container. Measure is 70MB - max age is the default value

**Comment by disaster123** [comment]
OK it's related to the measure folder. If i delete it it uses 90MB like with the empty config.

**Comment by disaster123** [comment]
I‘ve now set the storage time to 1day and will check what happens

**Comment by EinEinfach** [comment]
how are the tests going?

**Comment by disaster123** [comment]
It's raining all day here... i can't really test. Until now the process is now consuming again 233MB of memory.

**Comment by EinEinfach** [comment]
Strange. Can you check your logfiles, do you have some entries at 00:00. At 00:00 o'clock happens data clean procedure, is that done without errors?

**Comment by disaster123** [comment]
Yes and no - keep in mind that used memory isn't freed again under linux. So if it stays constant at 220MB - this will be OK - still a waste of 100MB but OK.

Log says:
```
2023-10-12 23:59:59 DEBUG Recieved checksum:0xe1 Calced checksum: 0xe1
2023-10-13 00:00:00 INFO Backend: Cleaning meausered and calced data. Max age: 1 days
2023-10-13 00:00:00 DEBUG state data: 38364 rows will be droped
2023-10-13 00:00:00 DEBUG stats data: 1315 rows will be droped
2023-10-13 00:00:00 DEBUG props data: nothing to clean
2023-10-13 00:00:00 DEBUG online data: 1084 rows will be droped
2023-10-13 00:00:00 DEBUG calced_from_state data: 38375 rows will be droped
2023-10-13 00:00:00 DEBUG calced_from_stats data: 1315 rows will be droped
```
